### PR TITLE
[STONEO11Y-221] Auto-assign the Editor role to users in the grafana main org

### PR DESCRIPTION
**Pull request by amisstea**
This allows logged in users to manage dashboards via the Grafana UI.

### CODE REVIEW

**Review by hmariset — APPROVED** [review]

**Review by kim-tsao — APPROVED** [review]
/lgtm

**Review by srivickynesh — APPROVED** [review]
/lgtm

File: components/monitoring/grafana/base/grafana-app.yaml [hunk 1]
```diff
@@ -101,6 +101,7 @@ spec:
   config:
     users:
       viewers_can_edit: true
+      auto_assign_org_role: Editor
```
Comment (gbenhaim) [inline components/monitoring/grafana/base/grafana-app.yaml:104]: This won't be needed IMO. The `viewers_can_edit` should grant users the ability to edit dashboards and run custom queries.

Comment (amisstea) [reply components/monitoring/grafana/base/grafana-app.yaml:104]: Ok, I agree. There's an `example` dashboard that can be used as a starting point when trying to create a new dashboard. Closing this PR.

**Pull request closed without merging**